requirements.txt

```diff
@@ -1,6 +1,7 @@
-horovod[tensorflow]>=0.19.5
-tensorflow>=2.2.0
```

Comment:
why was this removed?

Reply:
It's now in the Dockerfile:
- tensorflow is obtained from the base image nvcr.io/nvidia/tensorflow:20.12-tf2-py3
- horovod is manually installed to have support for tf and pt (didn't manage to make it work in the requirements)

Reply:
- Can we add the docker instructions also in the readme?
- Can we document it in the installation in a note that we need to install horovod manually.

Reply:
- Done - see latest pushed changes! 
- Manually was the wrong word, it's just installed from the Dockerfile to be able to do some stuff with ldconfig around it.